ansible/playbook.yml

```diff
@@ -1 +1,9 @@
 ---
+- hosts: ewf-ami
+  become: true
+  roles:
+    - user-dir-basic-config
```

Comment:
Should the name of the role being more ewf specific, this role contains non user/directory tasks so it should likely be a whole EWF role or create multiple roles with specific tasks.

Reply:
Renamed to ewf-app-config